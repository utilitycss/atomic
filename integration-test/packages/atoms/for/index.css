@for $i from 1 to 3 {
  .size_$(i),
  .size_$(i)_sibling {
    font-size: $(i)em;
  }
}

@for $i from 1 to 3 {
  .calc$(i){
    font-size: calc($(i) * 10px);

    @for $j from 1 to 3 {
      .child_$(i)_$(j) {
        width: $(j)em;
      }

      &.modifier_$(i)_$(j) {
        border: $(j)px red solid;
      }
    }
  }
}
