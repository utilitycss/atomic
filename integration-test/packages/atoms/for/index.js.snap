module.exports = {
  foo_1: "ZlJtcH",
  bar_1: "ZXM7RA",
  foo_2: "pUJQIY",
  bar_2: "Z3doKH",
  foo_3: "aGea0b",
  bar_3: "t8cRql",
  size_1: "oPlD0W",
  size_1_sibling: "NRaoKv",
  lineHeight_1: "oi5tdf",
  lineHeight_1_sibling: "W1CcjX",
  size_2: "FCG5GQ",
  size_2_sibling: "H9WShg",
  lineHeight_2: "_7x3EPh",
  lineHeight_2_sibling: "WO37ya",
  size_3: "d8yso7",
  size_3_sibling: "uGM3oi",
  lineHeight_3: "_3Ft6ps",
  lineHeight_3_sibling: "NbUW8z",
  calc1: "Y9lrZy",
  child_1_1: "_8Urr0R",
  modifier_1_1: "e2agTq",
  anotherChild_1_1: "_5v4SSH",
  anotherModifier_1_1: "iS88X_",
  child_1_2: "vlAfUd",
  modifier_1_2: "PgUB93",
  anotherChild_1_2: "KjddK4",
  anotherModifier_1_2: "GZomQK",
  child_1_3: "utNk9X",
  modifier_1_3: "bwOLNP",
  anotherChild_1_3: "_5UlDUQ",
  anotherModifier_1_3: "vA0LhQ",
  calc2: "gPZ5T-",
  child_2_1: "SnZr3u",
  modifier_2_1: "tNdUvj",
  anotherChild_2_1: "RTwpt_",
  anotherModifier_2_1: "aiCnKv",
  child_2_2: "bUAlDZ",
  modifier_2_2: "lpuLLH",
  anotherChild_2_2: "k5RQse",
  anotherModifier_2_2: "CmX6W8",
  child_2_3: "MdYjYw",
  modifier_2_3: "YK_gQZ",
  anotherChild_2_3: "_2W8BKO",
  anotherModifier_2_3: "F3eX_U",
  calc3: "_75_9PU",
  child_3_1: "KCb6ES",
  modifier_3_1: "_2PGpjA",
  anotherChild_3_1: "Y66lQo",
  anotherModifier_3_1: "sHxrAr",
  child_3_2: "_4_kJbH",
  modifier_3_2: "eCH6U5",
  anotherChild_3_2: "cx5ofG",
  anotherModifier_3_2: "iAQisI",
  child_3_3: "LSjKrm",
  modifier_3_3: "RYF8kF",
  anotherChild_3_3: "XKZMXH",
  anotherModifier_3_3: "Btd_L7"
};